web/src/app/components/survey-list/survey-list.component.spec.ts

```diff
@@ -55,7 +55,8 @@ describe('SurveyListComponent', () => {
         /* tasks= */ Map()
       ),
     }),
-    /* acl= */ Map()
+    /* acl= */ Map(),
+    OnboardingStatus.DONE
```

Comment:
Can we just allow `null` here and omit this value when no in the setup flow?

Reply:
Sg, done.